react/README.md

```diff
@@ -72,4 +83,5 @@ To eliminate repetitive code, a custom renderer was built in `react/src/utils/te
 * Ideally only one component per file
 * Files and folders which export a component/view are written UpperCamelCase, with the same name as the actual exported component/view
 * Other files (like types.ts, helpers.ts etc) and folders (like providers, utils etc) are written in lowerCamelCase
-
+* Config constants should be UPPERCASE_SNAKES
+* GraphQL queries, mutation and subscription  **string**  have the format UPPERCASE_SNAKES_<QUERY|MUTATION|SUBSCRIPTION>
```

Comment:
Aren't all existing queries mutations etc. currently lowercase_snakes @pylipp ?

Reply:
althought maybe it doesn't matter because FE/BE is separate.

Reply:
The note here refers to an actual query string, not the definition in the schema (GraphQL query/mutation definitions are camelCase).